src/NServiceBus.Gateway.AcceptanceTests/NServiceBus.Gateway.AcceptanceTests.csproj

```diff
@@ -19,6 +19,23 @@
     <PackageReference Include="NServiceBus.Callbacks" Version="3.0.0-beta0003" />
     <PackageReference Include="NUnit" Version="3.7.1" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <PackageId>NServiceBus.Gateway.AcceptanceTests.Sources</PackageId>
+    <Description>Acceptance tests for NServiceBus gateway functionality</Description>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_PackageFiles Include="**\*.cs">
+      <BuildAction>Compile</BuildAction>
+      <PackagePath>content\App_Packages\NSB.GW.AcceptanceTests\;contentFiles\cs\any\NSB.GW.AcceptanceTests\</PackagePath>
+    </_PackageFiles>
+    <_PackageFiles Remove="**\obj\**\*.cs" />    
+    <_PackageFiles Remove="AssemblyInfo.cs" />
```

Comment:
There is no `AssemblyInfo.cs` file in this is project, so this line isn't needed.